internal/server/device/tpm.go

```diff
@@ -227,6 +227,9 @@ func (d *tpm) startVM() (*deviceConfig.RunConfig, error) {
 		return nil, fmt.Errorf("Failed to save swtpm state for device %q: %w", d.name, err)
 	}
 
+	// swtpm unfortunately isn't always immediately ready for QEMU...
+	time.Sleep(time.Second)
```

Comment:
I wonder, if there isn't a nicer way to make sure, that the swtpm is indeed ready. With this sleep, we always loose 1 second here. On the other hand, waiting 1 second might also not always be enough for the swtmp to become ready e.g. on a heavily loaded or very minimal system.

Reply:
I was somehow remembering our older implementation where we were passing an fd which would have been pretty problematic to monitor, but given that following a fix for a path length issue, we're now using a regular socket, we can just wait for it to exist (after ensuring it doesn't exist prior to starting up).